bot/cogs/information.py

```diff
@@ -213,7 +191,7 @@ def __init__(self, bot: Bot):
                 {custom_status}
                 **Member Information**
                 Joined: {joined}
-                Roles: {roles or None}
+                Roles: {roles}
```

Comment:
Why was `or None` removed? It's better to show `None` than an empty string IMO.

Reply:
Every user has the `Developer` role per default, and users with no roles cannot use this command- making the `or None` check rather redundant. If you want it to be re-added, I can easily do so, I just thought that it wasn't needed :)

Reply:
Users who haven't verified i.e. gotten past the checkpoint will have no roles.

Reply:
To be clear, moderators can use this command to get info about other users, including those that haven't passed the checkpoint.